### PR TITLE
Remove style on classes used for strutured data

### PR DESCRIPTION
**Pull request by benoitchantre**
Having structured data or not should not have an influence on the appearance of the site.
This PR removes the dependancy of the microformats classes without changing the appearance by using new classes.

## Description

Addresses issue #86.

## List of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] My code is tested.
- [x] I want my code added to WP Rig.

### CODE REVIEW

**Comment by mor10** [comment]
I like this idea, but I think we need to move it to `v2.1` because what you've started on is a rethink of how CSS is used in the theme. This falls in line with the larger scope of moving to some form of BEM/ITCSS/CSS modules approach which we will focus on in `v2.1`. At that point we can take what you've done here and expand on it to apply to every element and style rule in the theme.

File: assets/css/src/content.css [hunk 1]
```diff
@@ -22,18 +22,18 @@
 }
 */
 
-.hentry {
+article {
```
Comment (mor10) [inline assets/css/src/content.css]: Change this to target `.article`, not the `article` element.

**Comment by mor10** [comment]
We've decided to merge this, provided you fix the `article` rule at the top to `.article` and update the PR to resolve conflicts.

**Comment by benoitchantre** [comment]
Thank you. Ready to merge.

**Comment by mor10** [comment]
Need one more review and then we can merge.

**Comment by benoitchantre** [comment]
@felixarntz The `hentry` class is added by Core to [add microformat support](http://microformats.org/wiki/hentry). If we keep `.hentry` but rename `entry-title`, which is marked as required, we will have validation errors in Google Search Console. If `hentry` is removed, then we remove support of microformat.

Do we want to remove the support of microformats?
[Google recommends to usage of schema](https://developers.google.com/search/docs/guides/intro-structured-data) (as `JSON-LD`).

**Comment by felixarntz** [comment]
Thanks for clarifying. In that case I suggest we do that and modify article to entry, thus causing less unnecessary changes.

**Pull request merged**